src/Cache.php

```diff
@@ -215,35 +210,36 @@ protected function setValues($data, $expire)
             }
         }
 
-        return $failedKeys;
+        // FIXME: Where do we access failed keys from ?
+        return count($failedKeys) === 0;
```

Comment:
setValues used to return failed keys. 

Are we dropping this feature since we're PSR compatible now, or should we provide a getter ?

Reply:
If there's nothing about this situation in both PSR and it's meta then yes, dropping it.